🐛 [Bug]:  Cross compile to FreeBSD fails for internal/gopsutil

### Bug Description

Cross compiling to FreeBSD on debian&darwin fails with:

```
github.com/gofiber/fiber/v2/internal/gopsutil/load/load_bsd.go:59:14: undefined: invoke
github.com/gofiber/fiber/v2/internal/gopsutil/load/load_bsd.go:78:19: cannot use f.forks (type int) as type int64 in assignment
```



### How to Reproduce

Steps to reproduce the behavior:
1. Run `GOOS=freebsd GOARCH=amd64 go build -o app` on macOS

### Expected Behavior

Should compile successfully but fails.

### Fiber Version

 v2.35.0

### Code Snippet (optional)

```go
package main

import "github.com/gofiber/fiber/v2"
import "log"

func main() {
  app := fiber.New()

  // Steps to reproduce
  app.Get("/metrics", monitor.New(monitor.Config{Title: "app"}))
  log.Fatal(app.Listen(":3000"))
}
```


### Checklist:

- [X] I agree to follow Fiber's [Code of Conduct](../CODE_OF_CONDUCT.md).
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

--- a/internal/gopsutil/load/load_bsd.go
+++ b/internal/gopsutil/load/load_bsd.go
@@ -9,8 +9,11 @@ import (
 	"strings"
 	"unsafe"
 
+	"github.com/gofiber/fiber/v2/internal/gopsutil/common"
 	"golang.org/x/sys/unix"
 )
+
+var invoke common.Invoker = common.Invoke{}
 
 func Avg() (*AvgStat, error) {
 	return AvgWithContext(context.Background())
@@ -75,7 +78,7 @@ func MiscWithContext(ctx context.Context) (*MiscStat, error) {
 	if err != nil {
 		return nil, err
 	}
-	ret.ProcsCreated = f.forks
+	ret.ProcsCreated = int64(f.forks)
 
 	return &ret, nil
 }

